<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawToolsWidget</class>
 <widget class="QWidget" name="DrawToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>98</width>
    <height>141</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="atom_frame">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="schrodinger::sketcher::SetAtomWidget" name="set_atom_wdg" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="charge_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="explicit_h_btn">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add or remove explicit hydrogens</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../sketcher.qrc">
            <normaloff>:/icons/atom_explicit_H.svg</normaloff>
            <disabledoff>:/icons/atom_explicit_H_dis.svg</disabledoff>:/icons/atom_explicit_H.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">explicit_h_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="increase_charge_btn">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Increase Charge</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../sketcher.qrc">
            <normaloff>:/icons/atom_charge_plus.svg</normaloff>
            <disabledoff>:/icons/atom_charge_plus_dis.svg</disabledoff>:/icons/atom_charge_plus.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">charge_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="decrease_charge_btn">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Decrease Charge</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../sketcher.qrc">
            <normaloff>:/icons/atom_charge_minus.svg</normaloff>
            <disabledoff>:/icons/atom_charge_minus_dis.svg</disabledoff>:/icons/atom_charge_minus.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">charge_group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bond_frame">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="schrodinger::sketcher::ModularToolButton" name="bond_query_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="schrodinger::sketcher::ModularToolButton" name="stereo_bond1_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="single_bond_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Single Bond</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/bond_single.svg</normaloff>
          <disabledoff>:/icons/bond_single_dis.svg</disabledoff>:/icons/bond_single.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="schrodinger::sketcher::ModularToolButton" name="stereo_bond2_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="atom_chain_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Bond Chain - Drag on canvas to draw a chain of carbons</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../sketcher.qrc">
          <normaloff>:/icons/bond_chain.svg</normaloff>
          <disabledoff>:/icons/bond_chain_dis.svg</disabledoff>:/icons/bond_chain.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="schrodinger::sketcher::ModularToolButton" name="bond_order_btn">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bond_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::ModularToolButton</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/widget/modular_tool_button.h</header>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::SetAtomWidget</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/widget/set_atom_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="charge_group"/>
  <buttongroup name="explicit_h_group"/>
  <buttongroup name="bond_group"/>
 </buttongroups>
</ui>
